src/pid_ros.cpp

```diff
@@ -126,7 +129,7 @@ PidROS::getDoubleParam(const std::string & param_name, double & value)
 bool
 PidROS::initPid()
 {
-  double p, i, d, i_min, i_max;
+  double p, i, d, i_min, i_max = std::numeric_limits<double>::quiet_NaN();
```

Comment:
this only sets the value for `i_max` which I'm sure is not what you intended.

For proof see https://godbolt.org/z/vW8vx3r3j

Reply:
Thanks for saving the day!

Reply:
```suggestion
  double p, i, d, i_min, i_max;
  p = i = d = i_min = i_max = std::numeric_limits<double>::quiet_NaN();
```